packages/css-library/dist/stylesheets/utilities.css

```diff
@@ -2326,6 +2407,186 @@ through all possible variants
     padding-right: 2rem;
   }
 }
+@media all and (min-width: 75em) {
+  .desktop-lg\:vads-grid-container-medium-screen {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 768px;
+    padding-left: 1rem;
+    padding-right: 1rem;
+  }
+}
+@media all and (min-width: 75em) and (min-width: 64em) {
+  .desktop-lg\:vads-grid-container-medium-screen {
+    padding-left: 2rem;
+    padding-right: 2rem;
+  }
+}
+@media all and (min-width: 48em) {
+  .medium-screen\:vads-grid-container {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 64rem;
+    padding-left: 1rem;
+    padding-right: 1rem;
+  }
+}
+@media all and (min-width: 48em) and (min-width: 64em) {
+  .medium-screen\:vads-grid-container {
+    padding-left: 2rem;
+    padding-right: 2rem;
+  }
+}
+@media all and (min-width: 48em) {
+  .medium-screen\:vads-grid-container-card {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 10rem;
+    padding-left: 1rem;
+    padding-right: 1rem;
+  }
+}
+@media all and (min-width: 48em) and (min-width: 64em) {
+  .medium-screen\:vads-grid-container-card {
+    padding-left: 2rem;
+    padding-right: 2rem;
+  }
+}
+@media all and (min-width: 48em) {
+  .medium-screen\:vads-grid-container-card-lg {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 15rem;
+    padding-left: 1rem;
+    padding-right: 1rem;
+  }
+}
+@media all and (min-width: 48em) and (min-width: 64em) {
+  .medium-screen\:vads-grid-container-card-lg {
```

Comment:
I have observed that there are less utilities getting generated now than what we had before when we were using our custom breakpoint code. 

The reason I think this is happening is because the custom code did not take into account the utility settings that identified the utilities that we didn't want to be generated with responsive breakpoints. For example, text-decoration utilities:

https://github.com/department-of-veterans-affairs/component-library/blob/a001910995d60b7136513225c20dda0ff982a348/packages/css-library/src/utilities/settings.scss#L136-L143